webapp/src/main/webapp/news-details/components/ExoNewsEditPublishingDrawer.vue

```diff
@@ -157,6 +157,9 @@ export default {
       } else {
         this.disabled = true;
       }
+    },
+    audience(){
+      this.disabled = false;
```

Comment:
> why this is needed ?
This modification to detect the change of the audience value in case of update and prevent the disabling of the update button.